src/vs/platform/configuration/test/common/testConfigurationService.ts

```diff
@@ -24,14 +24,14 @@ export class TestConfigurationService implements IConfigurationService {
 		// This is a temporary fix and should be removed once xterm GPU rendering is working again.
 		if (configuration) {
 			if (configuration.integrated) {
-				configuration.integrated.gpuAcceleration = 'off';
+				configuration.integrated.gpuAcceleration = 'canvas';
```

Comment:
What's the difference between "canvas" and "off" and how do you expect that'll help the issues?

Reply:
I just thought that since the error is occurring when an attempt to read 'getRasterizedGlyph' on an object that is undefined, and the error mentions xterm-addon-canvas that I would try this option.

I'm in the process of figuring out how the terminal uses the provided settings to be configured. I wasn't able to locate any documentation besides for what is in the readme in the xterm.js repo, so I've mainly been reading through code to figure things out.